Assets/MRTK/Examples/Demos/UX/Dwell/ToggleDwellPressableButton.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Dwell
+{
+    /// <summary>
+    /// Example script to demonstrate a toggle button using dwell
+    /// This script uses _BorderWidth property of Mixed Reality Standard Shader
+    /// </summary>
+    [AddComponentMenu("Scripts/MRTK/Examples/ToggleDwellPressableButton")]
+    public class ToggleDwellPressableButton : BaseDwellPressableButton
+    {
+        [SerializeField]
+        private TextMeshPro dwellStatus = null;
+
+        [SerializeField]
+        private GameObject buttonBackground = null;
```

Comment:
Can we make this of type MeshRenderer, since all instance of this object invoke GetComponent<MeshRenderer>()?